backend/remote-state/gcloud/backend.go

```diff
@@ -58,11 +58,14 @@ func (b *Backend) configure(ctx context.Context) error {
 		return nil
 	}
 
-	data := schema.FromContextBackendConfig(ctx)
+	// ctx is a background context with the backend config added.
+	// Since no context is passed to RemoteClient.Get(), .Lock(), etc. we
```

Comment:
This comment seems to be truncated

Reply:
D'oh, thanks for noticing. I'll update my branch in a second :)